healthgovau/ask
# 395 support subtheme app integration

**Changes:**

- Add support for installing apps in subthemes.
- Add support for yarn or npm based React projects.

**Testing instructions:**

- Install new site: `drush si --existing-config -y`
- Enable Health Sample Content module: `drush en health_sample_content -y`
- Install sample content: `drush mim --all`
- Install sample app: `composer require healthgovau/sample-react-app`
- Follow prompts. Select default values when defaults exist.
- Go to `http://ask.docker.amazee.io/resources/rebel-base-calculator/app` and confirm the app is displayed.
